app/models/pawn.rb

```diff
@@ -6,4 +6,39 @@ def image
       '&#9823;'
     end
   end
+
+  def valid_move?(x, y)
+    dx = delta_x(x_position, x)
+    dy = delta_y(y_position, y)
+
+    return false if blocked?(x, y)
+
+    # pawn can't move horizontally
+    return false if dx != 0 && dy.zero?
+
+    # pawn moves vertically 1 space
+    return true if dx.zero? && dy == 1
+    return false if color == 'white' && y_position < y
+    return false if color == 'black' && y_position > y
+    false
+  end
+
+  def capture_move?(x, y)
+    x_diff = (x_position - x).abs
+    y_diff = (y_position - y).abs
+    captured_piece = square_occupied?(x, y)
+    return true if x_diff == 1 && y_diff == 1
+    return false if captured_piece.blank?
+    # return false if captured_piece.color ==
+  end
+
+  def first_move?(y)
+    (color == 'white' && y_position == 6) || (color == 'black' && y_position == 1)
```

Comment:
This line checks to see if the piece is in its starting position, but it doesn't do anything with the check.  You can add `return false unless` to the beginning of the line, and that should make it return if it's not what you expected it to be.  That might however increase the line length beyond what Rubocop wants (120 characters I think?), so if you want, you can add some parenthesis in order to move the condition onto multiple lines:

```ruby
return false unless (
  (color == 'white' && y_position == 6) || (color == 'black' && y_position == 1)
)
```